module/common/RunLogger.cpp

```diff
@@ -0,0 +1,37 @@
+/**
+ * @file RunLogger.cpp
+ * @brief 走行中の各種値可視化のためのcsvファイル作成に関するクラス
+ * @author bizyutyu keiya121
+ */
+#include "RunLogger.h"
+
+RunLogger::RunLogger() {}
+
+void RunLogger::addTolog(int _Brightness, int _RightPwm, int _leftPwm, int _r, int _g, int _b)
```

Comment:
LogにするかLogsにするかは任せます

```suggestion
void RunLogger::addToLogs(int _Brightness, int _RightPwm, int _leftPwm, int _r, int _g, int _b)
```

Reply:
有留くんに対してではないけども同じところなのでここに。
_RightPwm と _leftPwm でアッパーキャメルとローワーキャメルが混ざってるので統一をお願いします。